### PR TITLE
Add LTR and LTC Tokens

### PR DESCRIPTION
**Pull request by SlightlyCircuitous**
*Fixes https://github.com/Cockatrice/Magic-Token/issues/226
*Adds entries for 5 new tokens and set lines for 5 reprints from LTR 
*Adds entries for 5 new tokens and set lines for 10 reprints from LTC

I currently have "The Ring" listed with type Emblem and have it reverse related to any card that says "The ring tempts you." as its referred to as an emblem on the "The Ring Tempts You" reminder card. I have listed said reminder card with type Token and related it to "The Ring" so that "The Ring" can make the reminder card if needed. Please feel free to change the types around if you think something fits better.

### CODE REVIEW

**Comment by ebbit1q** [comment]
has the entire set been spoiled yet?

**Comment by SlightlyCircuitous** [comment]
As far as I can tell, yes. All the tokens that we expect to have are up, the card counts line up with the expected number of cards on the wiki, and Scryfall has both LTR and LTC listed on the front page as new additions (granted, they don't say 'full preview' though).

**Comment by SlightlyCircuitous** [comment]
Also, something I just noticed while looking at the wiki is that the Halfling token has printed type 'Tolkien Creature' which is how it is listed on Scryfall (doesn't show up in a search for t:token). My script picked that up when making the entry, but I assume we would want the regular "Token Creature" in the xml.

File: tokens.xml [hunk 1]
```diff
@@ -10024,11 +10243,13 @@ This creature can't be enchanted.</text>
             <reverse-related count="x">Marut</reverse-related>
             <reverse-related count="x">Master of Ceremonies</reverse-related>
             <reverse-related>Meet and Greet "Sisay"</reverse-related>
+            <reverse-related count="2">Mines of Moria </reverse-related>
```
Comment (ebbit1q) [inline tokens.xml:10246]: why is there a space at the end?

**Review by ebbit1q — COMMENTED** [review]
Thank you for adding these!

**Pull request merged**